### PR TITLE
Add AWS Security Lake alert visualization in Wazuh dashboard

### LINKED ISSUE #6477: Configuring Amazon security lake documentation
Hello Team,

On testing the Amazon security lake configuration for 4.6.0 Apha using this [documentation](https://documentation-dev.wazuh.com/v4.6.0-alpha1/cloud-security/amazon/services/supported-services/security-lake.html#amazon-security-lake), A few issues were noted.

1. The documentation did not state that the time required for the module to finish executing may vary depending on the number of messages in the AMZ SQS Queue before events can be seen from the dashboard.
2. The documentation did not include steps on how to view events on the Wazuh dashboard

1. Visualizing the alerts.

You can visualize the alert data in the Wazuh dashboard. To do this, go to the Security events module and add the filters in the search bar to query the alerts:

- Apply filter Filter rule.groups: amazon_security_lake

image

`Note: The change above (Visualizing  the alerts) should be appended at the end of the document`

2.  Note:  Depending on the number of messages in the AWS SQS Queue, this module might take some time to finish its execution before alerts are displayed on the  Wazuh dashboard.

`The change above should be append right below the image `

### PR DESCRIPTION
**Pull request by fdalmaup**
## Description

Closes #6477. It adds a new section to the Amazon Security Lake page describing how to visualize the alerts generated by the module in the Wazuh dashboard. Also, highlights in a note how the events fetching time depends on the number of messages present in the queue.

### Docs building
- [x] Compiles without warnings.
### Code formatting and web optimization
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

File: source/cloud-security/amazon/services/supported-services/security-lake.rst [hunk 1]
```diff
@@ -213,3 +215,12 @@ Authentication
 
 
 More information about the different authentication methods can be found in the :ref:`Configuring AWS credentials <amazon_credentials>` documentation.
+
+Visualizing alerts in Wazuh dashboard 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once you set the configuration and restart the manager, you can visualize the Amazon Security Lake alerts in the Wazuh dashboard. To do this, go to the ``Security events`` module and, for better visualization, apply the filter ``rule.groups: amazon_security_lake``:
```
Comment (GGP1) [inline source/cloud-security/amazon/services/supported-services/security-lake.rst:222]: We should use bold for user interface elements

```suggestion
Once you set the configuration and restart the manager, you can visualize the Amazon Security Lake alerts in the Wazuh dashboard. To do this, go to the **Security events** module and, for better visualization, apply the filter ``rule.groups: amazon_security_lake``:
```

**Review by GGP1 — CHANGES_REQUESTED** [review]

**Review by GGP1 — APPROVED** [review]
LGTM

File: source/images/aws/security-lake-1.png
Comment (Selutario) [inline source/images/aws/security-lake-1.png:1]: According to previous reviews, this red box should probably be `#3585F9`:
- https://github.com/wazuh/wazuh-documentation/pull/6694#discussion_r1396226482

**Review by Selutario — CHANGES_REQUESTED** [review]
All good, just a small detail.

**Review by Selutario — APPROVED** [review]

File: source/cloud-security/amazon/services/supported-services/security-lake.rst [hunk 2]
```diff
@@ -185,7 +185,9 @@ Set the configuration inside the section ``<subscriber type="security_lake">``.
 After setting the required parameters, restart the Wazuh manager to apply the changes:
 
 .. include:: /_templates/common/restart_manager.rst
-Please note that the module's time of execution varies depending on the number of notifications present in the queue. If the ``<interval>`` value is less than the required time of execution, the :ref:`Interval overtaken<interval_overtaken_message>` message will be displayed in the ``ossec.log`` file.
+
+.. note::
+    The module's time of execution varies depending on the number of notifications present in the queue. This directly influences the time you have to wait to see every alert on the Wazuh dashboard. If the ``<interval>`` value is less than the required time of execution, the :ref:`Interval overtaken<interval_overtaken_message>` message will be displayed in the ``ossec.log`` file.
```
Comment (javimed) [inline source/cloud-security/amazon/services/supported-services/security-lake.rst:190]:
```suggestion

   The module execution time varies depending on the number of notifications in the queue. This affects the time to display alerts on the Wazuh dashboard. If the ``<interval>`` value is less than the execution time, the :ref:`Interval overtaken <interval_overtaken_message>` message appears in the ``ossec.log`` file.
```

File: source/cloud-security/amazon/services/supported-services/security-lake.rst [hunk 3]
```diff
@@ -213,3 +215,12 @@ Authentication
 
 
 More information about the different authentication methods can be found in the :ref:`Configuring AWS credentials <amazon_credentials>` documentation.
+
+Visualizing alerts in Wazuh dashboard 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once you set the configuration and restart the manager, you can visualize the Amazon Security Lake alerts in the Wazuh dashboard. To do this, go to the **Security events** module and, for better visualization, apply the filter ``rule.groups: amazon_security_lake``:
```
Comment (javimed) [inline source/cloud-security/amazon/services/supported-services/security-lake.rst:222]:
```suggestion
Once you set the configuration and restart the manager, you can visualize the Amazon Security Lake alerts in the Wazuh dashboard. To do this, go to the **Security events** module. Apply the filter ``rule.groups: amazon_security_lake`` for an easier visualization.
```

**Review by javimed — COMMENTED** [review]
I'm leaving some suggestions for your consideration.

**Review by javimed — APPROVED** [review]

**Pull request merged**